.github/workflows/base-containers.yaml

```diff
@@ -0,0 +1,79 @@
+name: Build Deploy Containers
+
+on:
+
+  # Always have a base image ready to go - this is a nightly build
+  schedule:
+    - cron: 0 3 * * *
```

Comment:
Does Github have a preferred time to run nightly builds, or is 3AM just a nice time to do it?

Reply:
No you can choose whatever time you like!

Reply:
And I actually looked at this one... https://researchapps.github.io/cron/... just because !

Reply:
Of course you did! :D  That is some interesting data. The correlation with UTC times makes sense, though.